tfx/extensions/google_cloud_big_query/example_gen/executor.py

```diff
@@ -65,9 +68,18 @@ def _BigQueryToExample(pipeline: beam.Pipeline, exec_properties: Dict[str, Any],
   """
   project = utils.parse_gcp_project(exec_properties['_beam_pipeline_args'])
   converter = _BigQueryConverter(split_pattern, project)
+  bq_kwargs = json.loads(exec_properties['custom_config'])
```

Comment:
Please name the variable more readable, e.g., `big_query_kwargs`

Reply:
done